index.bs

```diff
@@ -1017,6 +1017,20 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="constructors">Make non-critical function parameters optional</h3>
```

Comment:
Should we call talk about method and arguments instead? That would match IDL and might be clearer.

Reply:
The naming, while not the best - is for consistency with the previous section. Whole document level terminology aligning is probably better suited in another PR since we jump back and forth quite a bit. I didn't want two consecutive sections to use different terms.

Reply:
Filed #195.

Reply:
Thanks!